Plugins/Public/mobiledocking_plugin/Main.cpp

```diff
@@ -522,13 +591,10 @@ bool UserCmd_Process(uint client, const wstring &wscCmd)
 
 			//Force the docked ship to launch. The teleport coordinates have been set by the previous method
 			const CLIENT_DATA& dockedShipData = mobiledockClients[iDockedClientID];
-			JumpToLocation(iDockedClientID, dockedShipData.carrierSystem, dockedShipData.carrierPos, dockedShipData.carrierRot);
+			mobiledockClients[client].iDockingModulesAvailable++;

```

Comment:
this should probably happen when the ship actually gets jettisoned, rather than immediately

Reply:
Fixed.